docs/user-guide/how_to_use_stable_preempt_plugin.md

```diff
@@ -0,0 +1,201 @@
+# Stable Preempt Plugin User Guide
+
+## Background
+When we need to enable elastic training or serving, preemptible job's pods can be preempted or back to running repeatedly, if no cool down time set, these pods can be preempted again after they just started for a short time, this may cause service stability dropped.
+So we add "stablepreempt" plugin to ensure preemptible job's pods can run for at least some time set by user.
+
+## Environment setup
+
+### Install volcano
+
+Refer to [Install Guide](../../installer/README.md) to install volcano.
+
+### Update scheduler configmap
+
+After installed, update the scheduler configuration:
+
+```shell
+kubectl edit configmap -n volcano-system volcano-scheduler-configmap
+```
+
+Register `stablepreempt` plugin in configmap while enable `preempt` action
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: volcano-scheduler-configmap
+  namespace: volcano-system
+data:
+  volcano-scheduler.conf: |
+    actions: "enqueue, allocate, preempt, backfill"
+    tiers:
+    - plugins:
+      - name: priority
+      - name: gang
+      - name: conformance
+      - name: stablepreempt
```

Comment:
I strongly suggest change another plugin name instead of `stablepreempt`. 
It seems current preempt is not stable without this plugin:)

Reply:
How about "preempt-protected"?
or any other suggestions for the new plugin name?

Reply:
Let's name it `cdt` plugin which means cool down time or `cdp` which means cool down period.  It can be used not only in preemption but also in reclaim as well.

Reply:
@william-wang @flyhighzy I don't recommend using this name. There may be multiple ways to cool down victim tasks, not only by time interval. The name limits the functionality of the plugin and makes the functionality of this plugin slightly thin. How about `vcp` which means victim cooldown protection or `ecp` which means evicted cooldown protection.

Reply:
@Jason-Liu-Dream I noticed your [issue](https://github.com/Jason-Liu-Dream/volcano/issues/7), but maybe you need to put issue in the right repo :).   I think it sounds reasonable, but may need to discuss more details.

So what's your opinions? @william-wang @Thor-wl

Reply:
@Jason-Liu-Dream Would you clariy some other vailid cool down ways besides  the time/period?

Reply:
> @Jason-Liu-Dream I noticed your [issue](https://github.com/Jason-Liu-Dream/volcano/issues/7), but maybe you need to put issue in the right repo :). I think it sounds reasonable, but may need to discuss more details.
> 
this issue is put here because our inner version implements a demo of this function, we want to combine your design, and retry to design our implements.

Reply:
> @Jason-Liu-Dream Would you clariy some other vailid cool down ways besides the time/period?

We think cool down reason maybe need to consider time periods， eviction times， pod height load pressure,  block messages, message events and so on and different business scenarios have different requirements.

Reply:
@Jason-Liu-Dream Thanks for the explaining the different cases for the cool down operation.  If possible you can elaborate on pod height load pressure, block messages, message events. 
 
@flyhighzy I think we can refine the name to `clp` which means cool down protection. It can cover most of use cases so far :)

Reply:
sure, I think so :)

Reply:
> @Jason-Liu-Dream Thanks for the explaining the different cases for the cool down operation. If possible you can elaborate on pod height load pressure, block messages, message events.
>  @william-wang 
pod height load pressure: Sometimes we need to preempt some pods, but we don't want to preempt it when this pod is at height load pressure, height load pressure means a large scope of influence after stopping service in this pod. And we don't know how long this will last. so cooldown time does not work.
block messages: similar to above, sometimes message block in queue means there is a lot of stress to deal with in this service. Preempting is not a good idea.
message events: In batch tasks, we can record progress. Assuming that a task has N stages, and each stage supports breakpoints to continue working, then the best plan is to complete the goal of the previous stage, send a message event, and then let this task be preempted so that not cause excessive waste of resources. This is very common in pipeline jobs such as video transcoding.